charts/celeborn/README.md

```diff
@@ -40,17 +41,59 @@ When you want to test the template rendering, but not actually install anything.
 There are two ways to render templates. It will return the rendered template to you so you can see the output.
 
 - Local rendering chart templates
+
 ```shell
 helm template --debug ../celeborn
 ```
+
 - Server side rendering chart templates
+
 ```shell
 helm install --dry-run --debug --generate-name ../celeborn
 ```
-More details in [Helm Install](https://helm.sh/docs/helm/helm_install/)
-The chart can be customized using the following [celeborn configurations](https://celeborn.apache.org/docs/latest/configuration/#important-configurations)
-Specify parameters using `--set key=value[,key=value]` argument to `helm install`
+
+More details in [Helm Install](https://helm.sh/docs/helm/helm_install/).
+The chart can be customized using the following [celeborn configurations](https://celeborn.apache.org/docs/latest/configuration/#important-configurations).
+Specify parameters using `--set key=value[,key=value]` argument to `helm install`.
 
 ## Documentation
 
-For additional details on deploying the Celeborn Kubernetes Helm chart, please refer to the [Celeborn on Kubernetes](https://celeborn.apache.org/docs/latest/deploy_on_k8s/) documentation
+For additional details on deploying the Celeborn Kubernetes Helm chart, please refer to the [Celeborn on Kubernetes](https://celeborn.apache.org/docs/latest/deploy_on_k8s/) documentation.
+
+## Values
```

Comment:
let's remove this table at the current stage

Reply:
> let's remove this table at the current stage

@pan3793  Ok, already removed.